test/templates/common.py

```diff
@@ -0,0 +1,62 @@
+import subprocess
+import sys
+import re
+import os
+
+_me = os.path.basename(__file__)
+v = 0
+
+
+def set_basename(file):
+    global _me
+    _me = file
+
+
+def error(msg):
+    """
+    Print nicely-formatted error message and exit.
+    """
+    sys.stderr.write(f'{_me}: error: {msg}\n')
+    sys.exit(1)
+
+
+def run(cmd, stdin=None):
+    if v:
+        print(f"{_me}: running command: {' '.join(cmd)}")
```

Comment:
Maybe in the future we want to add some logging configuration to a file or something using Python logging facilities is this get more common. I guess this is okay for now because it's only here.

Reply:
Yes, using `logging` may make sense but I'm a bit weak on this. If you lead the way, I'll follow.

Reply:
I can do it later if needed. Just let this be like this for now.